stages/2-provider-infra-setup/infra-setup

```diff
@@ -96,10 +96,10 @@ kubectl apply -f oep-e2e/litmus/prerequisite/docker-secret.yml -n litmus
 node_name=$(kubectl get node | awk {'print $1'} | head -n 4 | tail -n 1)
 kubectl label nodes $node_name kubernetes.io/arch="amd64"
 kubectl label nodes $node_name kubernetes.io/os=linux
+
 ## Installing heapster components on the cluster for node monitoring
-git clone https://github.com/kubernetes-sigs/metrics-server.git
-sed -i -e '/args:/ a\          - --kubelet-insecure-tls' metrics-server/deploy/kubernetes/metrics-server-deployment.yaml
-kubectl apply -f  metrics-server/deploy/kubernetes/
+kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.6/components.yaml
+
```

Comment:
You need to download the file (https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.6/components.yaml) first and then run the `sed -i -e` line and then apply the yaml.